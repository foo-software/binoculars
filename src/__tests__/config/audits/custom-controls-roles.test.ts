import auditCustomControlsRoles from '../../../config/audits/custom-controls-roles';

describe('custom-controls-roles audit', () => {
  it('should match snapshot', () => {
    expect(auditCustomControlsRoles('en')).toMatchSnapshot();
  });
});
