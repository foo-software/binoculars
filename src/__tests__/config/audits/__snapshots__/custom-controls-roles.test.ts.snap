// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`custom-controls-roles audit should match snapshot 1`] = `
Object {
  "description": "Custom interactive controls have appropriate ARIA roles. [Learn more](https://web.dev/custom-control-roles/).",
  "title": "Custom controls have ARIA roles",
}
`;
